### DISCUSSION TITLE
Is it possible for /sysid to find all records with the  supplied sys_id not just the first one?

### DISCUSSION THREAD

**Discussion by codaroma** [arnoudkooi/SN-Utils#552] [Q&A]
I have found that it is quite normal to have multiple records in the same instance with the same sys_id so it would be super handy if the /sysid command found all of them. For example the sa_pattern table and the sa_pattern_draft table both re-use the same sys_ids.

**Comment by arnoudkooi** [comment]
I did not check, but would expect `sa_pattern_draft` to be the extended table of `sa_pattern` so it should return `sa_pattern_draft`?

**Reply by codaroma** [reply]
it's not
![image](https://github.com/user-attachments/assets/1850054d-a0f4-452e-ad92-e149b74f8f9c)
